types/constants.go

```diff
@@ -42,11 +42,12 @@ var (
 
 	// Oak hardfork constants. Oak is the name of the difficulty algorithm for
 	// Sia following a hardfork at block 135e3.
-	OakHardforkBlock BlockHeight
-	OakDecayNum      int64
-	OakDecayDenom    int64
-	OakMaxRise       *big.Rat
-	OakMaxDrop       *big.Rat
+	OakHardforkBlock        BlockHeight
+	OakDecayNum             int64
+	OakDecayDenom           int64
+	OakMaxRise              *big.Rat
+	OakMaxDrop              *big.Rat
+	OakHardforkTxnSizeLimit = uint64(65536) // 64 KiB
```

Comment:
Convention for sizes is to use base 10. It's not really relevant, but given that our blocks are 2e6, transactions should probably be 64e3